esy.json

```diff
@@ -4,14 +4,15 @@
   "license": "MIT",
   "version": "3.5.0",
   "dependencies": {
-    "ocaml": " >= 4.2.0 < 4.9.0",
-    "@opam/ocamlfind": "*",
+    "@opam/dune": "*",
     "@opam/menhir": " >= 20170418.0.0",
-    "@opam/utop": " >= 1.17.0 < 2.3.0",
     "@opam/merlin-extend": " >= 0.4",
-    "@opam/result": "*",
     "@opam/ocaml-migrate-parsetree": "*",
-    "@opam/dune": "*"
+    "@opam/ocamlfind": "*",
+    "@opam/result": "*",
+    "@opam/utop": " >= 1.17.0 < 2.3.0",
+    "@reason-native/rely": "^3.1.0",
```

Comment:
Rely should be a dev dependency, and there should be another empty file at the root called `reason-test`, and then you can add a field to "esy" field called `buildDev` which does `dune build -p reason,rtop,reason-dev`. This way, not everyone that depends on reason has to depend on rely.